project/Build.scala

```diff
@@ -1147,6 +1147,10 @@ object Build {
            |  > git submodule update --init
         """.stripMargin)
     }
+
+    // If contents .vscode do not exist yet, initialize them with the contents of .vscode-template/
```

Comment:
I would say "Copy default configuration from .vscode-template unless configuration files already exist in .vscode"